GoogleCloudPlatform/vertex-ai-samples
# chore(egen): follows new template, fixes typos, K80-->T4, REGION-->LOCATION etc.

<br>
This PR makes the following updates to the `multicontender_vs_champion_deloyment_method.ipynb` notebook:

- Change REGION to LOCATION
- Remove IS_TESTING condition at cleaning step
- Add Colab enterprise link
- Add Colab specific steps and other updates according to the new notebook template
- Remove future tense
- Contract words (like do not --> don't etc.)
- Fix the position of a sub-section (for "Save the model artifacts")
- Replace K80 with T4 in the text about GPUs
- Fix the typo "modesl"
- Moves the clean up step for yaml file to the last section
- Removes unnecessary code highlights (using "``")
- Handles the cases for Vertex AI resources
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>
